config/locales/en.yml

```diff
@@ -380,3 +349,10 @@ en:
           - Name
           - Signature
           - Date
+  start_page:
```

Comment:
Even though scoping isn't working through our `<%= component(...) %>` tags, we should try to format the `en.yml` to do the following so that scoping could work:

Reply:
Cool, I will change. I had initially went down that road but because the view file is so minimal in this case it felt like you weren't really saving anything.